docs/cloud-native-security/cloud-nat-sec-posture-dashboard.asciidoc

```diff
@@ -24,10 +24,17 @@ The Cloud Posture dashboard shows:
 [[cloud-nat-sec-posture-dashboard-UI]]
 === Cloud Posture dashboard UI
 
-The first row of cards (Cloud Posture Score, Failed Findings, and Open Cases) summarizes your overall cloud security posture (CSP) by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the CSP of an individual Kubernetes cluster.
+The top section of the dashboard summarizes your overall cloud security posture (CSP) by aggregating data from all monitored Kubernetes clusters. Three summary cards on the left of this section show Clusters Evaluated, Resources Evaluated, and Failed Findings. You can click *Resources Evaluated* or *Failed Findings* to open the <<findings-page, Findings page>> grouped by resources or filtered for failed findings, respectively. The remaining summary cards show your overall compliance score, and total failed findings grouped by CIS section. Click any CIS section name to view failed findings from that section on the <<findings-page, Findings page>>.
```

Comment:
Optional nit: This is a big chunk of text, especially on the rendered preview page, which can slow down readability. Maybe it could be broken into multiple paragraphs, or a bulleted list that identifies and describes each section of the dashboard. But nothing needs immediate attention, more of a thought for future revisions.

Reply:
In addition to what Joe mentioned here, I think a new image would break this up as well. I found myself scrolling to the image above to verify what was described here, and I'm not sure I followed 100%.